src/pseudopeople/interface.py

```diff
@@ -155,10 +155,10 @@ def generate_decennial_census(
     :param seed: An integer seed for randomness.
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
-    :param year: The year (format YYYY) to include in the dataset. Must be a decennial
-        year (e.g. 2020, 2030, 2040). Will raise a ValueError if there is no data for
-        this year. If None is provided, data for all years are
-        included in the dataset.
+    :param year: The year in which the census was conducted (format YYYY, e.g., 2030). Must be a decennial
+        year (e.g., 2020, 2030, 2040). Will raise a `ValueError` if there is no data for
```

Comment:
- Was this supposed to render as monospace? I think it renders as italics.
- Are you planning to make the formatting change on all the other parameters in a future PR? e.g. state, right below this, also contains the word "ValueError"

Reply:
I used italics to match how it looks in the function signatures and type hints. If we did double backticks instead, that would render like the code blocks in the rest of the docs.

Reply:
Gotcha, makes sense. It's kind of annoying that the Error names in the `:raises:` section below render as bold instead of italic, but I think italics is fine as long as we're consistent.

Reply:
> * Are you planning to make the formatting change on all the other parameters in a future PR? e.g. state, right below this, also contains the word "ValueError"

Yes, the descriptions of the `state` parameter also need changing, but I wanted to separate that out into a different PR.

> It's kind of annoying that the Error names in the :raises: section below render as bold instead of italic, but I think italics is fine as long as we're consistent.

Yeah, I noticed the inconsistent format of the errors, too, but they're rendered automatically, so it's probably not worth trying to figure out how to change them.

Ok, if you think italics look good and it's ok to be consistent within the API Reference even if it doesn't match the monospace in the rest of the docs, I'll keep the italic formatting for code on this page. The one exception right now is the rendering of `pseudopeople.NO_NOISE` at the top of the page, which is explicitly rendered in monospace on the `api_reference/noising/index.rst` page. Do you think we should change that to be italics to match the docstrings and function signatures?

Reply:
Also, do you think we should change the name of the `api_reference/noising/` directory to `api_reference/dataset_generation/` or something?

Reply:
> Yes, the descriptions of the state parameter also need changing, but I wanted to separate that out into a different PR.

Sounds good.

> Do you think we should change that to be italics to match the docstrings and function signatures?

No, I think we should use monospace for code in general; I don't mind using italics in docstrings for consistency with the function signature/typehint but my vote is to restrict that to docstrings.

> do you think we should change the name of the api_reference/noising/ directory to api_reference/dataset_generation/ or something?

Ideally, all of our URLs would match up with the page titles, but I don't consider this very important. And, it is always a big pain from a Git perspective if other work is going on. So it seems like doing this as a doc hotfix in the quiet period after release might be the best timing.

Reply:
One other option, just to see if this is appealing to you: we could say that italics are for _types_ (argument types and error types) and use monospace for `pseudopeople.NO_NOISE` everywhere.

Reply:
I guess None isn't technically a type, though.

Reply:
What about booleans? Those aren't types either, but in my other PR I changed those to italics in the docstrings to match the function signature. I mean, we could just use monospace in all the docstrings even if it won't match the automatically rendered stuff. I'm not sure which is better.

Reply:
Good point. As long as we are consistent within the docstrings, consistency with other places doesn't seem like a big deal.

Reply:
I checked in with Abie about it as well -- let's keep everything in the docstrings in italics for now.